CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
-### 0.5.1 (Next)
+### 0.6.1 (Next)
```

Comment:
Sure I will. Sorry for the misunderstanding. I've just started the open source involvement.  Thanks for your feedback.